.github/dependabot.yml

```diff
@@ -1,23 +1,23 @@
 version: 2
 updates:
   - package-ecosystem: npm
-    directory: "/"
+    directory: '/'
     schedule:
-      interval: daily
-      time: "04:00"
+      interval: weekly
```

Comment:
please keep it daily for dependencies and maybe move to weekly for dev dependencies